contributing/style.md

```diff
@@ -214,3 +214,36 @@ Leave an empty line between text and tables.
 | ---- | ---- |
 | 8:00 | **Opening** |
 ```
+
+## Links
+
+For articles that belong to the same directory, only link the article file name.
+
+```
+Like [statistical machine translation](statistical-machine-translation.md), neural machine translation is data-driven.
+```
+
+For articles that belong to another directory, add the directory name to the path.
+
+```
+She led early research on machine translation [quality estimation](/quality/quality-estimation.md).
```

Comment:
I would use the same example both here, above and below, to make it more obvious what the difference is.